Add support for node 10.x 11.x 12.x

amer8 opened the pull request:
See releases: https://nodejs.org/en/about/releases/

simenkid commented:
@amer8 There are failure in building with node v10, 11, and 12 (problems from levelup module). Could you check what's going on there?

amer8 commented:
@simenkid i think there's no simple fix. I would recommend a migration from `mosca` to `aedes` which doesn't have libzmq as dependency. By the way it's the successor project from the same author and should provide a better performance.

https://github.com/mcollina/aedes#acknowledgements
`This library is born after a lot of discussion with all Mosca users and how that was deployed in production. This addresses your concerns about performance and stability.`
